examples/pyspark_ml_autologging/logistic_regression.py

```diff
@@ -0,0 +1,22 @@
+from pyspark.ml.classification import LogisticRegression
```

Comment:
With these examples, we can quickly test changes for pyspark ml autologging and check how logged params/metrics/artifacts look like on MLflow UI.

Reply:
We have similar examples for sklearn autologging.